src/lambda_codebase/account_processing/configure_account_tags.py

```diff
@@ -35,6 +40,7 @@ def lambda_handler(event, _):
             event.get("tags"),
             organizations,
         )
+        EVENTS.put_event(detail=json.dumps({"tags": event.get("tags"), "account_id": event.get("account_id")}), detailType="ACCOUNT_TAGS_CONFIGURED", resources=[event.get('account_id')])
```

Comment:
Please reformat this. https://black.readthedocs.io/en/stable/getting_started.html can automatically do this for you